app/assets/javascripts/discourse/app/lib/sidebar/admin-sidebar.js

```diff
@@ -269,13 +270,30 @@ export default class AdminSidebarPanel extends BaseCustomSidebarPanel {
     const navMap = savedConfig || ADMIN_NAV_MAP;
 
     if (!session.get("safe_mode")) {
-      navMap.findBy("name", "plugins").links.push(...pluginAdminRouteLinks());
+      const pluginLinks = navMap.findBy("name", "plugins").links;
+      pluginAdminRouteLinks().forEach((pluginLink) => {
```

Comment:
Was this change necessary because we were doubling up with links at times or something?

Reply:
Yeah, that ajax call recalculates the cached function and could not prevent it, so this code must be temporarily idempotent